### DISCUSSION TITLE
Something went wrong! file an issue at https://tiny.one/readme-stats No GitHub APl tokens found Please add an env variable called PAT 1 with your GitHub APl token in vercel

### DISCUSSION THREAD

**Discussion by CoheeYang** [anuraghazra/github-readme-stats#4088] [Q&A]
i follow the video to deploy my own instance to show the private contribution (https://www.youtube.com/watch?v=n6d4KHSKqGk&t=107s)
env and a token are added ,but it failed and here is the link:
[![Anurag's GitHub stats](https://github-readme-stats-rouge-chi-16.vercel.app/api?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)

![微信图片_20241226141938](https://github.com/user-attachments/assets/6352c304-7a35-4aab-b09e-a6552975509b)

**Comment by tanvincible** [comment]
I faced the same issue. 
When I checked **Settings -> Environment Variables**, `PAT_1` wasn't listed there (I remember I added it while deploying). Try adding the `PAT_1` again, and redeploy.
